cli/cmd/diagnostics.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	adminHTTPPortName string = "admin-http"
+)
+
+type diagnosticsOptions struct {
+	wait time.Duration
+}
+
+func newDiagnosticsOptions() *diagnosticsOptions {
+	return &diagnosticsOptions{
+		wait: 300 * time.Second,
```

Comment:
For the metrics command, the timeout is 30s.  Is that what you meant here instead of 300?

Reply:
Yes, I intended to change it to 30s.

I had copied it from the `check` command which sets `wait` to 300 but forgot to change it. Thanks.